.github/workflows/maven-release.yaml

```diff
@@ -0,0 +1,31 @@
+# This workflow will put the project in our staging repo
+name: Releasing Project to maven
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  build:
+    if: endsWith(github.ref, "-lib")
+    env:
+      AWS_DEFAULT_REGION: us-east-1
+      AWS_REGION: us-east-1
```

Comment:
Are these environment variables used for anything btw?
```suggestion
```

Reply:
yeah if you dont have these set the unit tests fail 🤷🏻 something with initializing aws clients in unit test